changelog.md

```diff
@@ -7,6 +7,7 @@ Saves from 2.5 are not compatible with 3.0.
 * **[Campaign]** Ground units can now be transferred by road, airlift, and cargo ship. See https://github.com/dcs-liberation/dcs_liberation/wiki/Unit-Transfers for more information.
 * **[Campaign]** Ground units can no longer be sold. To move units to a new location, transfer them.
 * **[Campaign]** Ground units must now be recruited at a base with a factory and transferred to their destination. When buying units in the UI, the purchase will automatically be fulfilled at the closest factory, and a transfer will be created on the next turn.
+* **[Campaign AI]** Every 30 minutes the AI will plan a CAP, so players can customize their mission better. Default starting budget has increased to account for the increased need for aircraft.
```

Comment:
Budget hasn't changed (nor should it, since the default behavior hasn't changed).